.azure-pipelines-templates/deploy_aci.yml

```diff
@@ -1,3 +1,8 @@
+parameters:
+  secondaries:
+    image: ""
+    count: 0
```

Comment:
Yeah, I can do, would we like to keep the basic test I had in https://github.com/microsoft/CCF/commit/169cebda29b3e024d6b9643076202ac832d1a586 (Maybe with a better name since we're certainly not testing code update just yet)

Reply:
https://github.com/microsoft/CCF/pull/4861/commits/61cc900863e2d1f0463be959b8d93c18d337518f